.github/workflows/pnl-ci.yml

```diff
@@ -209,7 +220,7 @@ jobs:
 
     - name: Upload dist packages
       uses: actions/upload-artifact@v4
-      if: matrix.version-restrict == ''
+      if: matrix.version-restrict == '' && (! contains(matrix.python-architecture, 'x86') || ! contains(matrix.extra-args, 'not composition'))
```

Comment:
does:
```
overwrite: matrix.python-architecture == 'x86'
```
work?
the dist packages should be identical and it would be a bit easier to read

can you also add one sentence about the necessary adjustments to upload to the git commit description

Reply:
It looks like `overwrite` won't allow expressions, only true/false, unless I can't figure how to do it correctly. `overwrite: true` can still work and would currently only affect the x86. If we needed to add other jobs later without overwriting, we'd have to change the name anyway